04-Enumerable/Enumerable/Task.cs

```diff
@@ -322,9 +338,32 @@ public class Task {
         ///    {(1/1/2010, 10)  , (2/2/2010, 10), (3/3/2010, 10) } => { 30, 0, 0, 0 }
         ///    {(1/1/2010, 10)  , (4/4/2010, 10), (10/10/2010, 10) } => { 10, 10, 0, 10 }
         /// </example>
-        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales) {
-            // TODO : Implement GetQuarterSales
-            throw new NotImplementedException();
+        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales)
+        {
+            var grouped = sales.GroupBy(item => (item.Item1.Month - 1)/3, (key, list) => new Tuple<int, double>(list.Sum(item => item.Item2), list.Average(item => item.Item1.Month))).ToArray();
+
+            var result = new int[4];
+
+            foreach (var item in grouped)
+            {
+                if (item.Item2 <= 3)
```

Comment:
item2 is average moth in the list, even if the tests pass. The solutions is not correct